asset-transfer-sbe/application-javascript/app.js

```diff
@@ -0,0 +1,354 @@
+				// The endorsement will be handled by Discovery, since the gateway was connected with discovery enabled.
+				console.log(`\n--> Submit Transaction: CreateAsset, ${assetKey} as Org1 - endorsed by Org1 and Org2`);
+				await contractOrg1.submitTransaction('CreateAsset', assetKey, '100', 'Tom');
+				console.log('*** Result: committed, now asset will only require Org1 to endorse');
+			} catch (createError) {
+				console.log(`*** Failed: create - ${createError}`);
+			}
+
+			await readAsset(assetKey, 100, org1, contractOrg1, contractOrg2);
+
+			try {
+				// Since the gateway is using discovery we should limit the organizations used by
+				// discovery to endorse. This way we only have to know the organization and not
+				// the actual peers that may be active at any given time.
+				console.log(`\n--> Submit Transaction: UpdateAsset ${assetKey}, as Org1 - endorse by Org1`);
+				transaction = contractOrg1.createTransaction('UpdateAsset');
+				transaction.setEndorsingOrganizations(org1);
+				result = await transaction.submit(assetKey, '200');
+				console.log('*** Result: committed');
+			} catch (updateError) {
+				console.log(`*** Failed: update - ${updateError}`);
+			}
+
+			await readAsset(assetKey, 200, org1, contractOrg1, contractOrg2);
+
+			try {
+				// Submit a transaction to make an update to the asset that has a key-level endorsement policy
+				// set to only all Org1 to make updates. The following example will not use the "setEndorsingOrganizations"
+				// to limit the organizations that will do the endorsement, this means that it will be sent to all
+				// organizations in chaincode endorsement policy. Org1 will endorse, Org2 will not endorse, however
+				// since the Org1 approves and it is the only one required, the transaction will be committed.
+				console.log(`\n--> Submit Transaction: UpdateAsset ${assetKey}, as Org1 - endorse by Org1 and Org2`);
+				transaction = contractOrg1.createTransaction('UpdateAsset');
+				result = await transaction.submit(assetKey, '300');
+				console.log('*** Result: committed - because Org1 was on the discovery list, Org2 did not endorse');
+			} catch (updateError) {
+				console.log(`*** Failed: update - ${updateError}`);
+			}
+
+			await readAsset(assetKey, 300, org1, contractOrg1, contractOrg2);
+
+			try {
+				// Again submit the change to both Organizations by not using "setEndorsingOrganizations". Since only
+				// Org1 is required to approve, the transaction will be committed.
+				console.log(`\n--> Submit Transaction: UpdateAsset ${assetKey}, as Org2 - endorse by Org1 and Org2`);
+				transaction = contractOrg2.createTransaction('UpdateAsset');
+				result = await transaction.submit(assetKey, '400');
+				console.log('*** Result: committed - because Org1 was on the discovery list, Org2 did not endorse');
+			} catch (updateError) {
+				console.log(`*** Failed: update - ${updateError}`);
+			}
```

Comment:
Isn't this the same scenario as prior code block? What additional value does this one provide?

Reply:
different creator, different client, just shows either org could submit the change if it goes to the owning org peer to endorse